Shared/AppDelegateAdaptor.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AppDelegateAdaptor.swift
+//  Fun Gen
+//
+//  Created by jules on 4/16/22.
+//
+
+import SwiftUI
+import Firebase
+
+class AppDelegateAdaptor: NSObject, UIApplicationDelegate {
```

Comment:
This file does not appear in the Xcode project at all and is thus unused, consider deleting it?

Reply:
Was in there by accident. Deleted.